src/components/20-molecules/file-upload/README.md

```diff
@@ -124,3 +125,7 @@ The attribute `onFileDrop` function executes, as soon as the user dropped one or
 ### onFileRemove
 
 The attribute `onFileRemove` function executes, as soon as the user removed a file from the file-upload.
+
+### allowedFileTypes
+
+In the `allowedFileTypes` attribute, You can declaire the allowed file types that an user can upload (ex: only pdf's are allowed). If the attribute is empty, every file type is allowed.
```

Comment:
In the allowedFileTypes attribute, `you` can `declare` the allowed file types that `a user` can upload...

Reply:
-> It's "a user" because "user," even though it starts with a vowel, starts with a consonant Y sound.